include/mbedtls/ssl.h

```diff
@@ -382,6 +388,28 @@ union mbedtls_ssl_premaster_secret
 
 #define MBEDTLS_PREMASTER_SIZE     sizeof( union mbedtls_ssl_premaster_secret )
 
+static inline int mbedtls_ssl_chk_buf( uint8_t *cur, const uint8_t *end, size_t need )
```

Comment:
It's a bit unfortunate that our identifier checking script doesn't allow a line break after `static inline`.